game/templates/game/forum_list.html

```
@@ -67,29 +67,56 @@ <h1 class="forum-title">Community <span class="gold">Forum</span></h1>
             </p>
 
             <form method="get" class="forum-sort-form">
-                <label for="forum-sort">Sort by</label>
+                <div class="forum-search-box">
+                    <input type="text"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)